src/FactorGraph01.jl

```diff
@@ -1273,8 +1273,8 @@ function getKDE(dfg::G, lbl::Symbol) where G <: AbstractDFG
 end
 
 function expandEdgeListNeigh!(fgl::FactorGraph,
-                              vertdict::Dict{Int,Graphs.ExVertex},
-                              edgedict::Dict{Int,Graphs.Edge{Graphs.ExVertex}})
+                              vertdict::Dict{Int,TreeClique},
+                              edgedict::Dict{Int,Graphs.Edge{TreeClique}})
```

Comment:
this like this will likely break when we drop Graphs.jl for tree, but thats okay for now.  Just leaving a breadcrum incase we come back here to see why things are breaking.